UOP1_Project/Assets/Scripts/UI/FadeManager.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class FadeManager : MonoBehaviour
+{
+	[Header("Fadeout Event")]
+	[SerializeField] private FadeChannelSO _fadeChannelSO = default;
+
+	[Header("Fadeout Canvas")]
+	public GameObject _fadeoutCanvas = default;
+
+	private Image _imageComponent;
```

Comment:
why grab the canvas when you could just grab the image itself?
```suggestion
	[SerializeField] private Image imageComponent;
```
Just drag the FadeOut Rectangle in to get the image
then you can also remove the Start() method